examples/Huggingface_Transformers/Transformers_handler.py

```diff
@@ -0,0 +1,106 @@
+from abc import ABC
+import json
+import logging
+import os
+
+import torch
+from transformers import AutoModelForSequenceClassification, AutoTokenizer
+
+from ts.torch_handler.base_handler import BaseHandler
+
+logger = logging.getLogger(__name__)
+
+
+class TransformersSeqClassifierHandler(BaseHandler, ABC):
+    """
+    Transformers text classifier handler class. This handler takes a text (string) and
+    as input and returns the classification text based on the serialized transformers checkpoint.
+    """
+    def __init__(self, ctx):
+        super(TransformersSeqClassifierHandler, self).__init__()
```

Comment:
Same comment as before.
In general, both initialize and instantiation happen around the same time during model load stage. But the initialize function is called with service context  which gives the handler information provided about the model during registration such as:

- model_name
- model_dir
- manifest
- batch_size
- gpu
- ts.version

So the handlers can make better decisions during initialize than in constructor.